app/routes.py

```diff
@@ -0,0 +1,413 @@
+from app import db
+from app.models.customer import Customer 
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request
+import datetime
+from datetime import timedelta 
+
+
+# Establishing due date
+NOW = datetime.datetime.now()
+DUE_DATE = NOW + timedelta(days=7)
+
+# Blueprints
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# Customer model helper functions
+def display_customer_info(customer):
+    return {
+        "id": customer.id,
+        "name": customer.name,
+        "postal_code": customer.postal_code,
+        "phone": customer.phone
+    }
+
+def customer_not_found(customer_id):
+    return {"message" : f"Customer {customer_id} was not found"}
+
+# Video model helper function
+def video_not_found(video_id):
+    return {"message" : f"Video {video_id} was not found"}
+
+
+
+# ********************************
+# *** customers endpoint  CRUD ***
+# ********************************
+@customers_bp.route("", methods=["GET"])
+def get_all_customers():
+    """Lists all existing customers
+    and details about each customer
+    """
+
+    customers = Customer.query.all()
+    if customers is None:
+        return jsonify("Not Found"), 404
+
+    customer_response = []
+    for customer in customers:
+        customer_response.append(display_customer_info(customer))
+
+    return jsonify(customer_response), 200
+
+
+@customers_bp.route("/<customer_id>", methods=["GET"])
+def get_one_customer(customer_id):
+    """Returns details about a specific customer"""
+
+    if not customer_id.isnumeric():
+        return jsonify(None), 400
+    
+    customer = Customer.query.get(customer_id)
+    if customer == None:
+        return customer_not_found(customer_id), 404
```

Comment:
It's more _pythonic_ to use `is` when comparing values to `None` according to PEP8. 

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

https://www.python.org/dev/peps/pep-0008/#programming-recommendations

```suggestion
    if customer is None:
        return customer_not_found(customer_id), 404
```